wallet/test/run-wallet.c

```diff
@@ -176,6 +176,10 @@ void invoices_waitone(const tal_t *ctx UNNEEDED,
 void json_add_bool(struct json_result *result UNNEEDED, const char *fieldname UNNEEDED,
 		   bool value UNNEEDED)
 { fprintf(stderr, "json_add_bool called!\n"); abort(); }
+/* Generated stub for json_add_hex */
+void json_add_hex(struct json_result *result UNNEEDED, const char *fieldname UNNEEDED,
```

Comment:
fancy. where do these generated stubs get generated?

Reply:
`make update-mocks` will do that (but don't look behind the curtain) :wink: